README.md

```diff
@@ -9,7 +9,7 @@ IntelliJ IDEA / PhpStorm Symfony Plugin
 Key                  | Value
 -------------------- | --------------------
 Plugin Url           | https://plugins.jetbrains.com/plugin/7219
-ID                   | fr.adrienbrault.idea.symfony2plugin
+ID                   | fr.adrienbrault.idea.symfonyplugin
```

Comment:
`fr.adrienbrault.idea.symfonyplugin => fr.adrienbrault.idea.symfony2plugin`
we can never change the plugin id itself, else it would be a new plugin in jetbrains plugin repository :)